privateSBOMExchange/src/petra/lib/models/__init__.py

```diff
@@ -66,7 +62,9 @@ def __init__(self, field:str, value:str):
         """
         self.field_name = field
         self.field_value = value
+        self.plaintext_hash:str=""
```

Comment:
I think we should be computing this hash here when we first construct the node. We're also storing the hashes as bytes, since it's a little more space efficient.

Reply:
Also, it seems like `plaintext_hash` isn't being used anywhere yet. For the sake of merging this PR promptly, let's implement sameness proof generation/verification in a separate PR.